app/samples/udp_bcast/src/main.cpp

```diff
@@ -11,6 +11,7 @@
 #include <f_core/messaging/c_msgq_message_port.h>
 
 // Zephyr Includes
+#include <autoconf.h>
```

Comment:
what does autoconf do in this context?

Reply:
Nothing. IDE must've auto-added this for some reason